Backpack/src/main/java/net/skyscanner/backpack/util/BpkInterceptor.kt

```diff
@@ -0,0 +1,37 @@
+package net.skyscanner.backpack.util
+
+import android.graphics.drawable.ColorDrawable
+import android.os.Build
+import androidx.annotation.RequiresApi
+import io.github.inflationx.viewpump.InflateResult
+import io.github.inflationx.viewpump.Interceptor
+
+class BpkInterceptor : Interceptor {
+
+  @RequiresApi(Build.VERSION_CODES.M)
+  override fun intercept(chain: Interceptor.Chain): InflateResult {
+    val result = chain.proceed(chain.request())
+    val view = result.view()
+
+    if (view != null && isBpkView(view.javaClass)) {
+      if (view.background == null) {
+
+        view.background = ColorDrawable(0x77ac650c)
```

Comment:
nit: Would be good to move `0x77ac650c` to a constant with a descriptive name. E.g. `COLOR_BROWN` or just a comment here

Reply:
I will add as comment, as colors file is generated atm.